_security-analytics/usage/correlation-graph.md

```diff
@@ -0,0 +1,50 @@
+
+## Acccessing the graph
+
+Begin by selecting **Security Analytics** in the Dashboards home page menu. Then select **Correlations** from the Security Analytics menu on the left side of the screen. The **Correlations** page is displayed.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph.png" alt="The correlation graph" width="85%">
+
+## Interpreting the graph
+
+The graph displays findings as nodes with colored borders expressing their severity level. A three-letter abbreviation inside the node indicates the log type. The lines that connect the findings represent the correlations between them. A heavy line indicates a strong correlation, while a light line shows a weaker connection.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph-detail.png" alt="The correlation graph" width="40%">
+
+## Using the graph
+
+You can control which findings are displayed in the graph by filtering by severity level, log type, and time filter. The time filter controls the findings that appear in the graph by limiting the dates on which the findings were generated and, therefore, the correlations made between them.
+* Use the **Severity** dropdown menu to select which findings appear in the graph by their severity level. The number beside the menu name shows how many severity levels are being shown in the graph.
+* Use the **Log types** dropdown menu to select which log types to show on the graph. The number beside the menu name shows how many log types are being shown in the graph.
+* Select **Reset filters** to return the dropdown menus to their default, showing all items.
+* Use the time filter to set the the date range and show only those findings that were generated within that certain span of time. Select **Refresh** to bring the current number of findings up to date.
+
+You can drill down in the graph to look at correlations associated with a specific finding by selecting the finding in the graph. The graph then changes to show only the selected finding along with the constellation of findings correlated to it.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-drill-dn.png" alt="Zooming in on a specific finding in the graph" width="40%">
+
+After drilling down in the graph, informational cards for each of the findings appear in the right-hand margin of the screen. The selected finding appears first at the top of the cards and the correlated findings are listed below it in order of their correlation relevance, represented by a correlation score.
```

Comment:
In the **Correlation** panel? (instead of in the right-hand margin of the screen).